src/app/helpers/json/deepGet.test.js

```diff
@@ -5,6 +5,11 @@ const fixtureData = {
   bar: {
     baz: 456,
   },
+  nothing: {
+    nada: false,
+    thingsJonSnowKnows: 0,
+    zilch: '',
```

Comment:
```suggestion
    foobar: {
        barfoo: {
            zilch: '',
        }
    }
```

Can we try one nested to ensure it survives multiple passes of the reduce.